packages/components/inputs/toggle-input/src/toggle-input.styles.ts

```diff
@@ -0,0 +1,209 @@
+// TODO: @redesign cleanup
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import {
+  designTokens,
+  type ThemeName,
+} from '@commercetools-uikit/design-system';
+import type { TToggleInputProps } from './toggle-input';
+
+type NewThemeProps = {
+  isNewTheme: boolean;
+  theme: ThemeName;
+};
+
+type TStyledLabelProps = Pick<TToggleInputProps, 'isDisabled' | 'size'> &
+  NewThemeProps;
+type TStyledSpanProps = Pick<TToggleInputProps, 'size'> & NewThemeProps;
+
+const track = {
+  test: {
+    small: {
+      width: '29px',
+      height: '12px',
+    },
+    big: {
+      width: '56px',
+      height: '24px',
+    },
+  },
+  default: {
+    small: {
+      width: '32px',
+      height: '16px',
+    },
+    big: {
+      width: '64px',
+      height: '32px',
+    },
+  },
+};
+
+const thumb = {
+  test: {
+    small: {
+      diameter: '18px',
+      shift: '3px',
+      hoverAreaWidth: '4px',
+    },
+    big: {
+      diameter: '32px',
+      shift: '4px',
+      hoverAreaWidth: '8px',
+    },
+  },
+  default: {
+    small: {
+      diameter: '13px',
+      shift: '2px',
+    },
+    big: {
+      diameter: '26px',
+      shift: '3px',
+    },
+  },
+};
+
+const labelSizeStyles = (props: TStyledLabelProps) => css`
+  height: ${track[props.theme][props.size].height};
+  width: ${track[props.theme][props.size].width};
+`;
+
+const getThumbSize = (props: TStyledSpanProps) =>
+  thumb[props.theme][props.size].diameter;
+
+const getThumbShift = (props: TStyledSpanProps) =>
+  `${props.isNewTheme ? '-' : ''}${thumb[props.theme][props.size].shift}`;
+
+const getNewThemeTransformation = (props: TStyledSpanProps) =>
+  `calc(${thumb.test[props.size].shift} * 2 + ${
+    track.test[props.size].width
+  } - ${thumb.test[props.size].diameter})`;
```

Comment:
Nit: this is a bit hard to read. Can you extract the values into variables and assign the variables in the template string? The string then should be rendered in one line (more readable).

Reply:
Sure, great point, done here 3222e06cb56ad3fc1f4df9d38856320a0f3e4867